src/Maths/math.constants.ts

```diff
@@ -3,11 +3,32 @@
  * @ignorenaming
  */
 export const ToGammaSpace = 1 / 2.2;
+
 /**
  * Constant used to convert a value to linear space
  * @ignorenaming
  */
 export const ToLinearSpace = 2.2;
+
+/**
+ * Constant Golden Ratio value in Babylon.js
+ * @ignorenaming
+ */
+ export const PHI = (1 + Math.sqrt(5)) / 2;
+
+ /**
+ * Constant one third of the square root of 3
+ * @ignorenaming
+ */
+  export const THIRDR3 = Math.sqrt(3) / 3;

```

Comment:
I wonder if this should be in math.isovector only ? not sure this is reusable enough ?

Reply:
Done